### PR TITLE
Fixes spanish typos and translation errors

### PR DESCRIPTION
**Pull request by fagossa**

### CODE REVIEW

**Comment by SethTisue** [comment]
@heathermiller @dickwall do we know any Spanish speakers who could review this?

**Comment by SethTisue** [comment]
I mentioned this on scala/scala/scala-user on gitter, hoping to attract a reviewer or two.

File: es/tutorials/tour/default-parameter-values.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ En Java, uno tiende a ver muchos métodos sobrecargados que solamente sirven par
...
+Existen realmente dos constructores aquí; uno que toma otro mapa y uno que toma una capacidad y un factor de carga. Los constructores tercero y cuarto están ahí para premitir a los usuarios de la clase <code>HashMap</code> crear instancias con el valor por defecto que probablemente sea el mejor para ambos, el factor de carga y la capacidad.
 
-Más problemático es que los valores usados para ser por defecto están tanto en la documentación (Javadoc) y en el código. Mantener esto actualizado es fácil de olvidar. Un típico patrón utilizado para no cometer estos errores es agregar constantes públicas cuyo valor será mostrado en el Javadoc:
+Más problemático es que los valores usados para ser por defecto están tanto en la documentación (Javadoc) como en el código. Mantener ambos actualizado es dificil. Un patrón típico utilizado para no cometer estos errores es agregar constantes públicas cuyo valor será mostrado en el Javadoc:
```
Comment (rockneurotiko) [inline es/tutorials/tour/default-parameter-values.md]: "Mantener ambos actualizado" must be changed to "Mantener ambos actualizados"

File: es/tutorials/tour/abstract-types.md [hunk 2]
```diff
@@ -65,4 +65,4 @@ Es necesario notar que generalmente es posible transformar un tipo abstracto en
       println("content = " + buf.element)
     }
  
-Nótese que es debido usar [variance annotations](variances.html) aquí; de otra manera no sería posible ocultar el tipo implementado por la secuencia concreta del objeto retornado por `newIntSeqBuf`. Además, existen casos en los cuales no es posible remplazar tipos abstractos con tipos parametrizados.
+Nótese que es necesario usar [variance annotations](variances.html) aquí; de otra manera no sería posible ocultar el tipo implementado por la secuencia concreta del objeto retornado por `newIntSeqBuf`. Además, existen casos en los cuales no es posible remplazar tipos abstractos con tipos parametrizados.
```
Comment (miguel-vila) [inline es/tutorials/tour/abstract-types.md]: `remplazar` must be changed to `reemplazar` (note the double e)

File: es/tutorials/tour/anonymous-function-syntax.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ o sin parámetros:
 
     () => { System.getProperty("user.dir") }
 
-Esiste también una forma simple para escribir los tipos de las funciones. A continuación se muestran los tipos de las tre funciones escritas anteriormente:
+Existe también una forma simple para escribir los tipos de las funciones. A continuación se muestran los tipos de las trés funciones escritas anteriormente:
```
Comment (miguel-vila) [inline es/tutorials/tour/anonymous-function-syntax.md]: No need for an accent in `trés`, just `tres`

File: es/tutorials/tour/automatic-closures.md [hunk 4]
```diff
@@ -26,9 +26,9 @@ Para aclarar un poco esto aquí se muestra un ejemplo:
...
     }
 
-La función `whileLoop` recibe dos parámetros `cond` y `body`. Cuando la función es llamada, los parámetros reales no son evaluados en ese momento. Pero cuando los parámetros son utilizados en el cuerpo de la función `whileLoop`, las funciones nularias creadas implicitamente serán evaluadas en su lugar. Así, nuestro método `whileLoop` implementa un bucle tipo Java mediante una implementación recursiva.
+La función `whileLoop` recibe dos parámetros `cond` y `body`. Cuando la función es llamada, los parámetros reales no son evaluados en ese momento. Pero cuando los parámetros son utilizados en el cuerpo de la función `whileLoop`, las funciones nularias creadas implícitamente serán evaluadas en su lugar. Así, nuestro método `whileLoop` implementa un bucle tipo Java mediante una implementación recursiva.
 
-Es posible combinar el uso de [operadores de infijo y postfijo (infix/postfix)](operators.html) con este mecanismo para crear declaraciones más complejas (con una sintaxis agrdadable).
+Es posible combinar el uso de [operadores de infijo y postfijo (infix/postfix)](operators.html) con este mecanismo para crear declaraciones más complejas (con una sintaxis agradadable).
```
Comment (miguel-vila) [inline es/tutorials/tour/automatic-closures.md]: `agradadable` must be changed to `agradable`

File: es/tutorials/tour/default-parameter-values.md [hunk 5]
```diff
@@ -24,9 +24,9 @@ En Java, uno tiende a ver muchos métodos sobrecargados que solamente sirven par
       public HashMap(int initialCapacity, float loadFactor);
     }
 
-Existen realmente dos constructores aquí; uno que toma otro mapa y uno que toma una capacidad y un factor de carga. El tercer y cuarto constructores están ahí para premitir a los usuarios de la clase <code>HashMap</code> crear instancias con el valor por defecto que probablemente sea el mejor para ambos, el factor de carga y la capacidad.
+Existen realmente dos constructores aquí; uno que toma otro mapa y uno que toma una capacidad y un factor de carga. Los constructores tercero y cuarto están ahí para premitir a los usuarios de la clase <code>HashMap</code> crear instancias con el valor por defecto que probablemente sea el mejor para ambos, el factor de carga y la capacidad.
```
Comment (miguel-vila) [inline es/tutorials/tour/default-parameter-values.md]: `premitir` must be changed to `permitir`

File: es/tutorials/tour/default-parameter-values.md (hunk 1 above)
Comment (miguel-vila) [inline es/tutorials/tour/default-parameter-values.md]: `dificil` must be changed to `difícil`

**Comment by dickwall** [comment]
Thanks Miguel, how about we give this a week to try and attract a reviewer, and then merge it in either way. I do have someone in mind who might be able to give it a once over for correctness.

File: es/tutorials/tour/compound-types.md [hunk 6]
```diff
@@ -30,7 +30,7 @@ Ahora suponga que queremos escribir una función `cloneAndReset` la cual recibe
       cloned
     }
 
-La pregunta que surge es cuál es el tipo del parámetro `obj`. Si este fuera `Cloneable` entonces el objeto puede ser clonado mediante el método `clone`, pero no puede usarse el método `reset`; Si fuera `Resetable` podríamos resetearlo mediante el método `reset`, pero no sería posible clonarlo. Para evitar casteos (refundiciones) de tipos en situaciones como la descripta, podemos especificar que el tipo del objeto `obj` sea tanto `Clonable` como `Resetable`. En tal caso estaríamos creando un tipo compuesto; de la siguiente manera:
+La pregunta que surge es cuál es el tipo del parámetro `obj`. Si este fuera `Cloneable` entonces el objeto puede ser clonado mediante el método `clone`, pero no puede usarse el método `reset`; Si fuera `Resetable` podríamos resetearlo mediante el método `reset`, pero no sería posible clonarlo. Para evitar casteos (refundiciones, en inglés `casting`) de tipos en situaciones como la descrita, podemos especificar que el tipo del objeto `obj` sea tanto `Clonable` como `Resetable`. En tal caso estaríamos creando un tipo compuesto; de la siguiente manera:
```
Comment (sattvik) [inline es/tutorials/tour/compound-types.md]: What about `restablecerlo` rather than `resetearlo`?

**Comment by sattvik** [comment]
The patch with all of Miguel's suggestions LGTM.

**Comment by heathermiller** [comment]
Cool, thanks @sattvik!

**Comment by heathermiller** [comment]
I'll let @fagossa or anyone else willing to take @miguel-vila's & @sattvik's feedback and make a new PR out of it. (Anyone can fix these typos btw! Doesn't have to be only @fagossa)

**Pull request merged**